tests/test_sensors.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import logging
+
+def test_sensors(duthost, creds):
+    # Get platform name
+    platform = duthost.shell("show platform summary | grep Platform | awk '{print $2}'")['stdout']
```

Comment:
You can get platform from platform_info from gather_facts(). Understand that might be more heavier weighted. But using existing infrastructure could reduce maintenance cost in the future.

https://github.com/Azure/sonic-mgmt/blob/master/tests/common/devices.py#L132

Reply:
We could explicitly expose get_platform_info() from device.py if that helps. I want to dedup code.